src/api/lib/tasks/dev/sre.rake

```diff
@@ -0,0 +1,22 @@
+namespace :dev do
+  namespace :sre do
+    # Run this task with: rails dev:sre:configure
+    desc 'Configure the rails app to publish application health monitoring stats'
+    task :configure do
+      unless Rails.env.development?
+        puts "You are running this rake task in #{Rails.env} environment."
+        puts 'Please only run this task with RAILS_ENV=development'
+        return
+      end
+
+      RakeSupport.copy_example_file('config/options.yml')
+      options_yml = YAML.load_file('config/options.yml') || {}
```

Comment:
I got hit by [this Psych bug](https://bugs.ruby-lang.org/issues/17866) while testing the rake task. 

```suggestion
      options_yml = YAML.load_file('config/options.yml', aliases: true) || {}
```